src/sonic_ax_impl/mibs/vendor/cisco/ciscoPfcExtMIB.py

```diff
@@ -88,13 +91,12 @@ def _get_counter(self, oid, counter_name):
         :param counter_name: the redis table (either IntEnum or string literal) to query.
         :return: the counter for the respective sub_id/table.
         """
-        sai_id = self.oid_sai_map[oid]
```

Comment:
is oid_sai_map is being used or this can be removed ?

Reply:
Removed oid_sai_map as it is not required. The required information can be retrieved from oid_name_map.